src/components/profile/utils.js

```diff
@@ -42,6 +43,58 @@ export async function handleLoanMutationUpdates(
   }
 }
 
+/**
+ * Get an url for the profile page - loans, reservation, orderhistory or cart
+ * @param materialId
+ * @param materialType
+ * @returns {"/work/?type="|string}
+ */
+export function getWorkUrlForProfile({
+  workId = "",
+  pid = "",
+  materialId = "",
+  materialType = "",
+}) {
+  // workid is given
+  if (!isEmpty(workId)) {
```

Comment:
Egentlig bør rækkefølgen ved være
pid
faust
workId

For hvis vi har en pid, så har vi vel altid en `workId`, og udgaven må vel være prioriteret over værket... eller hvad?

Reply:
jo det har du nok ret i